app/components/DialogOverlay.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
```

Comment:
Lets either place this into `app/components/Util` or `app/components/UI` so that we don't have dangling files in `app./components`

Reply:
done